tests/validation/tests/v3_api/test_global_registry.py

```diff
@@ -0,0 +1,124 @@
+import pytest
```

Comment:
Moving forward, lets do validation tests as separate PRs so that they can be easily flagged for QA team to review

Reply:
Got it